.circleci/jobs/test-audius-cmd.yml

```diff
@@ -38,4 +38,4 @@ steps:
         - run:
             name: audius-cmd test
             no_output_timeout: 15m
-            command: . ~/.profile; audius-compose up -w && audius-cmd test
+            command: . ~/.profile; audius-compose up -w && sleep 60 && audius-cmd test
```

Comment:
I wasn't actually sure why either, but works with, doesn't work without, unfortunately was easier than trying to figure out what it was waiting on.

I think is ok for now since it is only one point where we add sleep.

Reply:
Oh this makes sense then.... it's b/c the stack is not healthy after doing audius-compose up.
Better to wait for some indication of health checks resolving